### PR TITLE
home screen description & guidelines

### PR DESCRIPTION
**Pull request by htcollier**
## Testing

- [ ] Tested on iOS 
- [ ] Tested on Android 

## PR Checklist

  **Reviewer:** Confirm the items below as you review

- [ ] PR is connected to issue(s)
- [ ] Tests are included to cover this change (when possible)
- [ ] No magic strings (All string unions follow the [Union -> Constant](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/develop/VAMobile/src/constants/common.ts) type pattern)
- [ ] No secrets or API keys are checked in
- [ ] All imports are absolute (no relative imports)
- [ ] New functions and Redux work have proper TSDoc annotations

## For QA

[Run a build for this branch](https://github.com/department-of-veterans-affairs/va-mobile-app/actions/workflows/on_demand_build.yml)

### CODE REVIEW

File: VAMobile/documentation/docs/Flagship design library/Templates/Home.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+* See the things that are likely relevant and valuable to them first
+* Get a consistent information structure for screen readers to tab through that is present even if load times lag
+* See things that are important to proper app functioning when necessary
+
+
+### Zones
+
+The VA Mobile app Home screen zones ensure that the most impactful content items are getting the proper placement in the screen's hierarchy while also handling a variety of user contexts and technical situations. 
+
+<img src="/workspaces/va-mobile-app/VAMobile/documentation/static/img/home/vamobile-home-zones.png" width="700" alt="Diagram of home screen zones for VA mobile app">
```
Comment (TimRoe) [inline VAMobile/documentation/docs/Flagship design library/Templates/Home.md:29]:
```suggestion
<img src="/workspaces/va-mobile-app/VAMobile/documentation/static/img/home/vamobile-home-zones.png" width="700" alt="Diagram of home screen zones for VA mobile app" />
```

File: VAMobile/documentation/docs/Flagship design library/Templates/Home.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
+---
+title: Home Screen
+sidebar_position: 4
+---
+
+The Home screen is the initial/default screen of the VA Health & Benefits mobile app. It sits at the top level of the hierarchical navigation (as a peer to the Health, Benefits and Payments category screens) and displays a personalized summary of a Veteran's current tasks and interactions with features in the VA mobile app. It also contains the primary entry point for the Veteran's Profile information (including App Settings).
+
+<iframe width="500" height="800" alt="image of VA mobile app Home screen" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Fdesign%2Fcdp7Be4UdYesq9fXeqaOgt%2FNavigation-2.0-Screen-Templates---%F0%9F%9A%A2-Shipped---VA-Mobile%3Fm%3Dauto%26node-id%3D7733-11182%26t%3DGsZrpZDn1qDXDTr0-1" allowfullscreen></iframe>
```
Comment (timwright12) [inline VAMobile/documentation/docs/Flagship design library/Templates/Home.md:8]:
```suggestion
<iframe width="500" height="800" title="image of VA mobile app Home screen" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Fdesign%2Fcdp7Be4UdYesq9fXeqaOgt%2FNavigation-2.0-Screen-Templates---%F0%9F%9A%A2-Shipped---VA-Mobile%3Fm%3Dauto%26node-id%3D7733-11182%26t%3DGsZrpZDn1qDXDTr0-1" allowfullscreen></iframe>
```

File: VAMobile/documentation/docs/Flagship design library/Templates/Home.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+#### Zone 3: About you:
+
+* **What it is:** A personalized, high-level snapshot of the individual Veteran at VA. 
+    * **Module appearance logic:** Fixed appearance.
+* **What lives in this zone:**
+    * Branch of Service/Veteran Status card (MVP)
+    * Disability rating & monthly payment (MVP)
+* **Guidelines for module inclusion:**
+Information is appropriate for inclusion on the home screen within About you when it:
+    * <span style="text-decoration:underline;">Is</span> or <span style="text-decoration:underline;">provides access to</span> information relating to who the individual Veterans *is within VA* as opposed to relating to a specific activity or task
```
Comment (timwright12) [inline VAMobile/documentation/docs/Flagship design library/Templates/Home.md:78]: Can we change these to some other type of emphasis? underlining non-links causes some a11y concerns

File: VAMobile/documentation/docs/Flagship design library/Templates/Home.md (hunk 1 above)
Comment (htcollier) [reply VAMobile/documentation/docs/Flagship design library/Templates/Home.md:29]: I need to able to control the width & height, so I used an img tag

File: VAMobile/documentation/docs/Flagship design library/Templates/Home.md (hunk 3 above)
Comment (htcollier) [reply VAMobile/documentation/docs/Flagship design library/Templates/Home.md:78]: yep

File: VAMobile/documentation/docs/Flagship design library/Templates/Home.md (hunk 1 above)
Comment (timwright12) [reply VAMobile/documentation/docs/Flagship design library/Templates/Home.md]: Can the image be resized instead?

**Review by timwright12 — APPROVED** [review]

**Pull request merged**